### ISSUE TITLE
Lost post votes or message from deleted post or thread

### ISSUE THREAD

**Issue by Visentinel** [HelloZeroNet/ZeroNet#2391]
### Step 1: Please describe your environment

  * ZeroNet version: any recent
  * Operating system: bug for zerotalk (windows 10)
  * Web browser: palemoon
  * Tor status: available
  * Opened port: yes
  * Special configuration: ____

### Step 2: Describe the problem:
People complain about not been able to post due to limit not getting raised so they go back and delete posts or entire threads then others people's posts get lost in the ether unable to manage them.

It also creates a continuity problem because posts that get deleted may cause threads to become incoherent and confusing to follow.

Limits maybe to low ? Or use a different method ?

**Comment by filips123** [comment]
People can ask @HelloZeroNet via ZeroMail to raise their limits.

**Comment by Crypto-Anarchist** [comment]
That's right! @Visentinel https://github.com/HelloZeroNet/ZeroMail/issues/new :eyes:

**Comment by Visentinel** [comment]
Someone was complaining that it took to long

But that doesn't resolve the problem that if someone deletes content your likes and comments get trapped and you would have to manually edit your content.json yourself and most people don't know how to
